WordPress/gutenberg
# Paste Handler: Try to fix pasting text with formatting

Fixes #63434

## What?

This PR fixes an issue where copying and pasting text that contains inline formatting would strip the formatting from the pasted text.

## Why?

I'm not sure what the root cause is, but it seems the problem occurs when the copied text contains formatted text at the end.

Also, when pasting multiple times, the formatted text seems to be pasted correctly either the odd or even number of times.

## How?
Removed the early return that was causing the problem.

We'll review what tests fail to ensure this doesn't introduce any unintended regressions.

## Testing Instructions

- Insert a Paragraph block and type your text.
- Apply inline formatting to the text.
- Select the text and copy it. Make sure to include the inline formatting at the end of the selection.
- Paste it into another paragraph.
- The formatting should be preserved.

## Screenshots or screencast 

### Before

https://github.com/user-attachments/assets/bc79467c-e385-457b-bd72-6c6ff78e2900

### After

https://github.com/user-attachments/assets/f8c1f06b-5ae9-450e-8c79-1c67090ffd8d
